src/components/study-pane.js

```diff
@@ -61,19 +61,20 @@ const useStyles = makeStyles((theme) => ({
             },
         },
     },
-    mapCtrlBottomLeft: {
+    mapCtrlBottomLeft: (theme) => ({
         '& .mapboxgl-ctrl-bottom-left': {
             transition: theme.transitions.create('left', {
                 easing: theme.transitions.easing.sharp,
                 duration: theme.transitions.duration.leavingScreen,
             }),
         },
-    },
+    }),
     table: {
         display: 'flex',
         flexDirection: 'column',
+        height: '100%',
```

Comment:
new rule needed ?

Reply:
Before this PR, we used to have two different classes:
`clsx('singlestretch-child', classes.table)`

The singlestretch-child rule was defined in the global index.css and contains height: '100%', but with emotion, I do not want to use this global class only for this, so I removed the reference to singlestretch-child and added the height rule directly in the styles.table. This updated rule is used only once, so that's just cleaner.